### PR TITLE
docs(store-devtools): Added documentation for new extension options for Store Devtools

### LINKED ISSUE #1519: docs: Add new extension options for Store Devtools
## I would be willing to submit a PR for the docs :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

Store Devtools now supports persisting, locking, pausing, along with whitelisting and blacklisting of actions. These should be added to the Instrumentation docs.

### PR DESCRIPTION
**Pull request by wesleygrimes**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?
Updated documentation on Store Devtools Config page as per #1519 

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #1519 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 1]
```diff
@@ -1,31 +1,81 @@
...
+    reorder: true, // drag and drop actions in the history list 
+    dispatch: true, // dispatch custom actions or action creators
+    test: true // generate tests for the selected actions
+},
+```
+
+## Example Object as provided in module imports
+
+```typescript
+@NgModule({
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]: Could you use the  `code-example` tag here? 😄 


```
<code-example header="app.module.ts">
codes come here
</code-example>
```

Comment (wesleygrimes) [reply projects/ngrx.io/content/guide/store-devtools/config.md]: @timdeschryver absolutely! I missed that.

Comment (wesleygrimes) [reply projects/ngrx.io/content/guide/store-devtools/config.md]: OK, this has been updated to use the tag as suggested.

**Comment by wesleygrimes** [comment]
@timdeschryver I agree with your changes, however, somehow I (embarassingly) deleted my local fork of this PR. I'm not sure how in GitHub to resolve this issue. Apologize, not trying to create additional work for anyone.

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 2]
```diff
@@ -1,31 +1,81 @@
...
+    test: true // generate tests for the selected actions
+},
+```
+
+## Example Object as provided in module imports
+
+<code-example header="app.module.ts">
+@NgModule({
+  ...
+  imports: [
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/store-devtools/config.md]: Indention should be 2 spaces on the next new line.

**Comment by brandonroberts** [comment]
@wesleygrimes no worries, just create a new fork and apply these changes there. Then open a new PR and we'll close this one.

**Comment by brandonroberts** [comment]
Closing in favor of #1533

**Pull request closed without merging**